package.json

```diff
@@ -18,14 +19,15 @@
     "@types/react-dom": "^16.9.8",
     "date-fns": "^2.16.1",
     "fhirpath": "^2.9.0",
-    "fqm-execution": "^0.3.1",
+    "fqm-execution": "git+https://github.com/projecttacoma/fqm-execution.git",
```

Comment:
Was having some trouble with permissions early on in this branch and I think this line is the result of trying to install fqm-execution in some unorthodox ways. Is this going to be a problem?

Reply:
Most likely you'll want to revert this. yeah. Looks like `fqm-execution` has been updated to `0.4.0`, so you could switch this line back then run an `npm update fqm-execution` which should grab the latest.